bundle/src/main/java/com/adobe/acs/commons/dam/impl/ReviewTaskAssetMoverHandler.java

```diff
@@ -367,29 +368,29 @@ private void moveAsset(ResourceResolver resourceResolver, AssetManager assetMana
                     if (exists) {
                         if (StringUtils.equals(asset.getPath(), destAssetPath)) {
                             log.info("Reviewed asset [ {} ] is already in its final location, so there is nothing to do.", asset.getPath());
-                        } else if(CONFLICT_RESOLUTION_REPLACE.equals(conflictResolution)) {
+                        } else if (CONFLICT_RESOLUTION_REPLACE.equals(conflictResolution)) {
                             assetManager.removeAsset(destAssetPath);
                             resourceResolver.commit();
                             assetManager.moveAsset(asset.getPath(), destAssetPath);
-                            log.info("Moved with replace [ {} ] ~> [ {} ] based on approval status [ " + status + " ]",
-                                    asset.getPath(), destAssetPath);
-                        } else if(CONFLICT_RESOLUTION_NEW_ASSET.equals(conflictResolution)) {
+                            log.info("Moved with replace [ {} ] ~> [ {} ] based on approval status [ {} ]",
```

Comment:
Actually:
`log.info("Moved with replace [ {} ] ~> [ {} ] based on approval status [ {} ]", asset.getPath(), destAssetPath, status)`

Should work.